### DISCUSSION TITLE
support webService ？

### DISCUSSION THREAD

**Discussion by xzy1990** [apache/apisix#6914] [Q&A] [answered]
Does apisix support the webService protocol  ？ If not，How to do in apisix？

**Comment by tzssangglass** [comment]
If you're talking about the webservice that uses the soap data format, as far as I know it doesn't support.

**Reply by xzy1990** [reply]
Do you konw someone who do this function

**Reply by tzssangglass** [reply]
I don't know.

**Reply by xzy1990** [reply]
So I think this piece should not be implemented in the upstream service

**Reply by tzssangglass** [reply]
I can't get your option

**Comment by moonming** [comment, marked as answer]
APISIX support SOAP now. 
Please try `body-transformer` plugin: https://apisix.apache.org/docs/apisix/plugins/body-transformer/
